### PR TITLE
Add rule for comments similar to `yamllint`

### PR DESCRIPTION
**Pull request by dawidmalina**

### CODE REVIEW

**Comment by dawidmalina** [comment]
closes #104

File: saltlint/rules/CommentHasSpaceRule.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import re
+
+
+class CommentHasSpaceRule(SaltLintRule):
+    id = '213'
+    shortdesc = 'Comment character should be always followed by at least one space.'
+    description = 'Comment character should be always followed by at least one space.'
+    severity = 'LOW'
+    tags = ['formatting']
+    version_added = 'v0.1.1'
```
Comment (jbouter) [inline saltlint/rules/CommentHasSpaceRule.py]:
```suggestion
    version_added = 'develop'
```

File: saltlint/rules/CommentHasSpaceRule.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Dawid Malinowski <dawidmalina@gmail.com>
+
+from saltlint.linter import SaltLintRule
```
Comment (jbouter) [inline saltlint/rules/CommentHasSpaceRule.py]: Pylint seems to fail on the order of importing. Please move `import re` above `from saltlint.linter import SaltLintRule`

**Comment by dawidmalina** [comment]
@roaldnefs, @jbouter can you check this PR? Feel free to update or modify code if needed.

**Pull request closed without merging**